go-controller/pkg/metrics/ovn_northd.go

```diff
@@ -73,6 +73,22 @@ var ovnNorthdCoverageShowMetricsMap = map[string]*metricDetails{
 	},
 }
 
+var ovnNorthdStopwatchShowMetricsMap = map[string]*stopwatchMetricDetails{
+	"ovnnb_db_run":    {},
```

Comment:
This is more of a question, but why does only `ovn_northd_loop` specify a `srcName`? `ovnControllerStopwatchShowMetricsMap` above had all key value pairs specify a `srcName`.

Reply:
The reason why only `ovn_northd_loop` has a `srcName` is that this metric is provided as `ovn-northd-loop` and so differs from the key (`ovn_northd_loop`). The other metric names are the same as the keys of this map.